### PR TITLE
fixes #9502, #9503 - verify errata exist before application

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
[test]

**Comment by ehelms** [comment]
[test]

File: app/controllers/katello/api/v2/system_errata_controller.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@
 module Katello
   class Api::V2::SystemErrataController < Api::V2::ApiController
     before_filter :find_system
+    before_filter :find_errata_ids
```
Comment (jlsherrill) [inline app/controllers/katello/api/v2/system_errata_controller.rb]: shouldn't this be :only => :apply  ?

Comment (stbenjam) [inline app/controllers/katello/api/v2/system_errata_controller.rb]: @jlsherrill  Ah, yes it should... updated

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**